### DISCUSSION TITLE
文件无法删除

### DISCUSSION THREAD

**Discussion by MiratPH** [galacean/discussion#97] [Editor] [answered]
上传较大的glTF文件报错失败后，编辑器行为异常，各种文件删不掉。例如图中已将所有项目删除，但仍显示占用500MB空间
![图片](https://github.com/galacean/discussion/assets/49740099/92c815f3-2da1-4c4d-8396-84dceb2b2de2)

**Comment by gz65555** [comment, marked as answer]
glTF 限制最多上传 50m 以内，最近会优化友好提示。删除项目占用空间问题已经修复。
